app/views/myprojects/edit.html.haml

```diff
@@ -1,6 +1,7 @@
 .page-header
-  %h1 Request Change for  App "#{App.find(@app.id).name}"
-
+  %h1 
+    Request Change for App "#{App.find(@app.id).name}"
```

Comment:
Wouldn't `@app.name` work instead of `App.find(@app.id).name`? It is one less database query.